docs/intro-to-eda.md

```diff
@@ -1,56 +1,53 @@
-    * For example, an appointment was created.
-* **Event (instance)**: a specific instance of a business action that has occurred  (note: event, message, and record can be seen interchangeably in various Kafka-related documentation).
-    * For example, an appointment was created for Karen to see an orthopedic specialist on 6/1/2023 at 10:30 am.
-
-# Learn More
-* [Event Streaming Patterns](https://developer.confluent.io/patterns/)
-* [Improving Veteran Benefit Services Through Efficient Data Streaming | Robert Ezekiel, Booz Allen Hamilton](https://www.confluent.io/events/kafka-summit-americas-2021/improving-veteran-benefit-services-through-efficient-data-streaming/)
-* [One Year In – Lessons Learned and Plans for the Future with Robert Ezekiel | Current 2022](https://www.confluent.io/resources/presentation/one-year-in-lessons-learned-and-plans-for-the-future/)
-* The Enterprise Event Bus Team offers consultations and is happy to answer questions. Reach out to us [link to contact page](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/get-support/).
-
-# Having Trouble?
-If you find something wrong with the documentation, didn’t find what you’re looking for, or have a question or suggestion, please [contact us](./get-support.md).
+When determining what information an event should contain, a guiding principle is that consumers need events to at least contain enough information to know whether the event is of interest to them, while minimizing the amount of sensitive data in the event payload. In general it is easier to add new pieces of information to a schema than it is to later remove it; thus the system should trend towards leaner events.
 
+##  Technology overview
+
+Enterprise Event Bus, like most other event-based systems in VA, is based on [Apache Kafka](https://kafka.apache.org/), an open-source distributed event streaming platform. It is considered the industry standard for handling real-time data feeds, and is capable of handling the kinds of large scale, high-throughput loads we would expect an enterprise-wide event bus in VA to be able to handle. In addition to being a proven technology, Kafka is already used in a production capacity by other teams in VA, such as the [Benefits Integration Platform](https://confluence.devops.va.gov/pages/viewpage.action?spaceKey=VAExternal&title=Benefits+Integration+Events).
+
+The Enterprise Event Bus uses [AWS Managed Streams for Kafka (MSK)](https://docs.aws.amazon.com/msk/index.html), a hosted version of Kafka that runs in the VA Enterprise Cloud. 
+
+_The following diagram illustrates how events pass through the different components of the Enterprise Event Bus system. View the diagram in [Github](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Diagrams/Data%20Flow.jpeg)._
+![Simple Flow EBus](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/95644573/61c8f134-7228-4735-b9df-c0e1985d9eaa)
 
```

Comment:
Similar to above, let's replace "View the diagram in Github" with `[View the full-sized diagram.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/95644573/61c8f134-7228-4735-b9df-c0e1985d9eaa)`

Reply:
Updated. Marking as resolved.